.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
```

Comment:
travis automatically picks up the version from `.nvmrc` - no need to over specify nd give ourselves more work. We do this in other repos too btw.

Reply:
See https://travis-ci.org/Skyscanner/backpack/builds/427659336#L432